src/utils/parseIntegrationTestRunnerOptions.ts

```diff
@@ -0,0 +1,74 @@
+export function parseIntegrationTestRunnerOptions(): {testCommand: string, skipBuild: boolean, debugMode: boolean, runtimeVersion: string|undefined} {
```

Comment:
Can we have a type for the return?

Reply:
Done. Though should I be explicitly exporting the type, since it's the return type of an exported function? And is there anything extra I'd need to put in index.ts

Reply:
No I dont think so.  Since its an NPM module its all JS by the time is ever used.  If it were to be used in the project elsewhere then the IDE should handle it for us.